<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DESKTOP-7B2OP92" name="tests.company_Admin.bdeo.CA_InterventionTests" tests="5" failures="0" timestamp="09 Nov 2018 10:31:07 GMT" time="471.380" errors="1">
  <testcase name="listOfInterventionForCA" time="255.251" classname="tests.company_Admin.bdeo.CA_InterventionTests"/>
  <testcase name="addInterventionForCA" time="157.205" classname="tests.company_Admin.bdeo.CA_InterventionTests"/>
  <testcase name="updateInterventionForCA" time="0.000" classname="tests.company_Admin.bdeo.CA_InterventionTests">
    <skipped/>
  </testcase> <!-- updateInterventionForCA -->
  <testcase name="cancelIntervention" time="0.000" classname="tests.company_Admin.bdeo.CA_InterventionTests">
    <skipped/>
  </testcase> <!-- cancelIntervention -->
  <testcase name="viewInterventionForCA" time="58.924" classname="tests.company_Admin.bdeo.CA_InterventionTests">
    <error type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
at java.util.ArrayList.rangeCheck(Unknown Source)
at java.util.ArrayList.get(Unknown Source)
at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementListHandler.invoke(LocatingElementListHandler.java:39)
at com.sun.proxy.$Proxy10.get(Unknown Source)
at pages.interventions.InterventionsPage.selectAgentFromList(InterventionsPage.java:201)
at steps.interventions.InterventionsPageSteps.searchByAgent(InterventionsPageSteps.java:103)
at tests.company_Admin.bdeo.CA_InterventionTests.viewInterventionForCA(CA_InterventionTests.java:191)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- viewInterventionForCA -->
</testsuite> <!-- tests.company_Admin.bdeo.CA_InterventionTests -->
